GEOparse/GEOTypes.py

```diff
@@ -726,9 +726,9 @@ def _get_object_as_soft(self):
             soft.append(self.database._get_object_as_soft())
         soft += ["^%s = %s" % (self.geotype, self.name),
                  self._get_metadata_as_string()]
-        for gsm in self.gsms.itervalues():
+        for gsm in self.gsms.values():
```

Comment:
Hi, I would use six library and the six.itervalues function for making compatibility. For an example see this commit: https://github.com/guma44/GEOparse/pull/8/commits/a979fa5dfa1f8fd5c086db330cdbce844827febe by tychobismeijer